sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -52,9 +52,15 @@ case class CatalogStorageFormat(
     properties: Map[String, String]) {
 
   override def toString: String = {
+    val maskedProperties = properties.map {
+      case (password, _) if password.toLowerCase == "password" => (password, "###")
```

Comment:
I think you should rename the variable "password" to just key. For a moment I thought that was the password. Same thing with url.

Also in the case of URL, do we really want to hide the URL completely? Can't we just hide the password?